test_extras/test_chemprop/chemprop_test_utils/default_models.py

```diff
@@ -0,0 +1,27 @@
+"""Functions for creating default chemprop models."""
+
+from molpipeline.estimators.chemprop.component_wrapper import (
+    MPNN,
+    BinaryClassificationFFN,
+    BondMessagePassing,
+    SumAggregation,
+)
+
+
+def get_classification_mpnn() -> MPNN:
```

Comment:
Rather name the function `get_binary_classification_mpnn`

Reply:
Done